### ISSUE TITLE
There is no snap respository in China and it  is very slow ,Could you provide ppa respository

### ISSUE THREAD

**Issue by KElli0t** [Genymobile/scrcpy#1010]
THe download speed  is about 30 k/s in china through the snap respository, could you provide the ppa respository  .

**Comment by KaKi87** [comment]
Hello @rom1v,
Any news on this ?
Thanks

**Comment by rom1v** [comment]
Nope, I don't maintain any packages, only the official release (zip for Windows and prebuilt scrcpy-server).

You can install the latest version following these steps on Linux: https://github.com/Genymobile/scrcpy/blob/master/doc/linux.md#latest-version

**Comment by KaKi87** [comment]
Actually, the picture clearly indicates that there is currently no way to install the latest version on Debian-based distros.

**Comment by rom1v** [comment]
On the first line of the `Latest Version" section:

> To install the latest release from `master`, follow this simplified process.
> …

**Comment by KaKi87** [comment]
Hello,
Any news on this ?
Thanks

**Comment by KaKi87** [comment]
Hi,

A few months ago, I started developing a tool named [dynapt](https://git.kaki87.net/KaKi87/dynapt) to automatically fetch updates through APT for packages that don't provide an APT repository.

As I've been upgrading scrcpy through it for quite some time now, I just published my first pre-release of it, which you can download and set up from [this tutorial](https://gist.github.com/KaKi87/f620788e9901abbfef4978eb7ad358b4).

For scrcpy specifically, replace the following configuration line :

```json
    "apps": []
```

With the following :

```json
    "apps": [
        {
            "name": "scrcpy",
            "github": {
                "repo": "ardevd/scrcpy-appimage"
            },
            "appimage": {
                "bin": "scrcpy"
            }
        }
    ]
```

In this case, it makes use of @ardevd's third-party AppImage builds, since no official DEB or AppImage is provided.

For questions or issues, please submit a comment on the tutorial page.

Thanks

---

cc. @yolowex (#5186), I know you solved your issue, but if you ever want something simpler, here it is !

**Comment by yolowex** [comment]
Thanks!
Your package seems fun to use!
